.github/workflows/documentation-tests.yml

```diff
@@ -55,6 +55,7 @@ jobs:
         run: |
           pip install --upgrade --index-url https://test.pypi.org/simple/ --extra-index-url https://pypi.org/simple/ pennylane-catalyst pennylane-lightning
           pip install -e .
+          # TODO: use 0.7.0 after updating all the documentation
```

Comment:
Does this need to happen in this PR or in a future PR?

Reply:
Future PR. I don't think we can fully clean all the documentation regarding jax for now, since Catalyst will never be up to run these doctests.